packages/mcp-workflow/src/routers/checkPropertiesFulfilledRouter.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { StateType, StateDefinition } from '@langchain/langgraph';
```

Comment:
`checkPropertiesFulfilledRouter` is just moving down into `mcp-workflow` wholesale from `mobile-native-mcp-server`, because it's generic and useful to all consumers.